app/models/course_template.rb

```diff
@@ -0,0 +1,15 @@
+class CourseTemplate < ActiveRecord::Base
```

Comment:
I know the project doesn't have these consistently but could you add a short comment describing what this is for?